kong/plugins/http-log/handler.lua

```diff
@@ -28,7 +29,7 @@ end
 local function parse_url(host_url)
   local parsed_url = url.parse(host_url)
   if not parsed_url.port then
-    if parsed_url.scheme == "http" then
+    if parsed_url.scheme == HTTP then
```

Comment:
I think both those strings (`http` and `https`) should not be stored in those constants, they serve no purpose since they are not reused anywhere else in the code.